packages/app/src/app/overmind/effects/live/clients.ts

```diff
@@ -65,13 +65,7 @@ export class CodeSandboxOTClient extends OTClient {
           })}`,
         });
 
-        // We make sure to not acknowledge the same revision twice
-        if (this.lastAcknowledgedRevision < revision) {
-          this.lastAcknowledgedRevision = revision;
-          this.serverAck();
-        } else {
-          this.resetAwaitSynchronized();
```

Comment:
We need this one to be able to save after a duplicate message. Otherwise this promise will never resolve and we're stuck on saving.